gcloud-java-contrib/README.md

```diff
@@ -7,6 +7,12 @@ Contents
 --------
 
  * [gcloud-java-nio](./gcloud-java-nio/): NIO Filesystem Provider for Google Cloud Storage.
+ * [gcloud-java-nio-example](./gcloud-java-nio-example/): How to add GCS NIO after the fact.
```

Comment:
OK, though I think a notion of an "examples" folder would not confuse people if there is only one example there however if we ever do add more examples calling the folder example is bit confusing.